src/tpt_reports/tpt_reports.py

```diff
@@ -167,9 +178,13 @@ def main() -> None:
     output_directory: str = validated_args["OUTPUT_DIRECTORY"]
     json_file_path: str = validated_args["JSON_FILE_PATH"]
 
-    # Set up logging
+    # Setup logging to central file
     logging.basicConfig(
-        format="%(asctime)-15s %(levelname)s %(message)s", level=log_level.upper()
+        filename=LOGGING_FILE,
+        filemode="a",
+        format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+        datefmt="%m/%d/%Y %I:%M:%S",
+        level=log_level.upper(),
```

Comment:
Why are these logging changes in this PR?  They don't have anything to do with the goal of this PR, do they?

Reply:
Resolved in commit 31438682e24e3b3ece8e773aa8f4e474fbcc82cd
Created issue https://github.com/cisagov/tpt-reports/issues/40

Reply:
@dav3r Just following up on these to see if there is anything we need to modify. Thanks for taking a look!

Reply:
I will resolve this conversation, though please note my related comment [here](https://github.com/cisagov/tpt-reports/pull/37#discussion_r1325109404).